admin-api/src/main/java/com/depromeet/franchise/service/FranchiseService.java

```diff
@@ -18,14 +19,21 @@ public class FranchiseService {
     public void saveFranchise(final CreateFranchiseRequest request) {
         validateDuplicatedFranchiseName(request.name());
 
-        final Franchise franchise = Franchise.builder()
+        final var franchise = Franchise.builder()
                 .name(request.name())
                 .imageUrl(request.imageUrl())
                 .build();
 
         franchiseRepository.save(franchise);
     }
 
+    @Transactional
+    public void modifyFranchiseImage(final Long id, final ModifyFranchiseImageRequest request) {
+         final var franchise = franchiseRepository.findById(id);
+         franchise.modifyImageUrl(request.imageUrl());
+         franchiseRepository.updateImage(franchise);
+    }
+
     private void validateDuplicatedFranchiseName(final String name){
         franchiseRepository.findByName(name).ifPresent(
```

Comment:
name으로 같은 이름을 가진 프랜차이즈를 조회하는데 ifPresent로 이미 존재한다면 바로 Exception을 보내도 괜찮지 않을까요??

isSameName() 이라는 행위가 이미 앞서 프랜차이즈를 DataBase에서 가져오고 ifPresent로 검증을 했다고 생각해요.

만약 그래도 isSameName()을 통해 검증을 하고 싶다면 아래와 같이 하는 건데, 이것도 어찌보면 당연하게 isSameName에서 if문이 true가 되어 exception을 반환하겠네요
```suggestion
        franchiseRepository.findByName(name).map(Franchise::isSameName);
```

Reply:
이건 수찬님 의견이 맞는 것 같아요! 굳이 IsSame으로 검증하지 않아도 되긴 하네요 ㅎㅎ